flake.nix

```diff
@@ -155,7 +164,43 @@
           default = {
             type = "app";
             program = "${rocBuild.roc-cli}/bin/roc";
+            meta = {
+              description = "Roc CLI";
+              mainProgram = "roc";
+            };
           };
         };
+
+        # test for nix/buildRocPackage.nix
+        checks.canBuildRocPackage =
+          let
+            helloWorldPackage = buildRocPackage {
+              inherit pkgs;
+              roc-cli = rocBuild.roc-cli;
+              linker = "legacy";
+              name = "helloworld";
+              optimize = true;
+              # use `src = ./myfolder;` for local usage
+              src = pkgs.fetchFromGitHub {
+                owner = "roc-lang";
+                repo = "examples";
+                rev = "main";
+                sha256 = "sha256-DqqkA5iASoK68XBFKv6Gbrso4687smKz8PqVUL2rRsE=";
+              };
+              entryPoint = "./examples/HelloWorld/main.roc";
```

Comment:
I've used hello world here instead and got it straight from the examples repo to avoid needing to sync changes to a local file.